### PR TITLE
Global Banner styles

### PR DESCRIPTION
**Pull request by aknobloch3**
Global Banner styles
- Added banner examples to Paragon
- Reformatted banners to have shared styles

### CODE REVIEW

**Comment by aknobloch3** [comment]
There is a bug in Firefox with allowing the content to scroll on the x-axis. So working on that... :(

**Comment by aknobloch3** [comment]
@jrauh @Nevren - Updated the global banner to have a container around it. Took away the `::after` because that was keeping Firefox from scrolling the banner on smaller screens.

Also, used a mixin to share the similar banner styles across the different banners we use.

File: src/sass/modules/banner/_banner.scss [hunk 1]
```diff
@@ -1,3 +1,51 @@
+$content-width: 1200px !default;
+$global-banner-height: 3.5rem;
+$global-banner-height_phone: 4rem;
+
+/* Shared Banner Styles
+  =========================================================================== */
+@mixin shared-banner {
```
Comment (jrauh) [inline src/sass/modules/banner/_banner.scss]: I'm not totally sure from the naming what a "shared" banner is opposed to global.

Comment (aknobloch3) [reply src/sass/modules/banner/_banner.scss]: Valid... Created a mixin to share styles across multiple banners. 

How about `default-banner`?

Comment (jrauh) [reply src/sass/modules/banner/_banner.scss]: Would that be different than `.banner`? Is there a `.banner` class? We just don't use "default" in other shared elements

Comment (aknobloch3) [reply src/sass/modules/banner/_banner.scss]: yeah. I guess that really is what it should be. Not sure why I decided to go the mixin approach?

**Pull request merged**